### ISSUE TITLE
Design/Implement basic homepage

### ISSUE THREAD

**Issue by xTrig** [UWCodeForce/student-net#51]

**Comment by niloc515** [comment]
Ive come up with a [rough draft]https://www.figma.com/file/nCRpPt1CTWnW4AdYe0UTec/Student-net-home-page-draft?node-id=5%3A2), I'm a beginner to Figma and would love some extra eyes and/or opinions on what I have.

Would love an extra set of eyes or opinions to help develop it further! (especially if you know more Figma than I do)

**Comment by gabrielople** [comment]
Hi, I can get assigned for this part.

**Comment by gabrielople** [comment]
> Ive come up with a [rough draft]https://www.figma.com/file/nCRpPt1CTWnW4AdYe0UTec/Student-net-home-page-draft?node-id=5%3A2), I'm a beginner to Figma and would love some extra eyes and/or opinions on what I have.
> 
> Would love an extra set of eyes or opinions to help develop it further! (especially if you know more Figma than I do)

Nice design! I think we can add a bit more features and information to this one. It doesn't have much yet, but it's a good start. We could just fix some design issues here and there. We could also change some sizing of the elements, add a bit more stuff to make it feel like an actual homepage, and etc. I'd be happy to work with you on creating a design for this homepage! :-)

**Comment by gabrielople** [comment]
Not really sure which design we're going for so I just made a low-fi wireframe on Figma. I finished the homepage but haven't started on the courses page because I'm not too sure about our functional requirements. For now, I'll be putting all our stuff in one Figma file so all our designs will be there! Here is the file: https://www.figma.com/file/MekzVSBx89UspTohZ43XDi/Student-Net---Code-Force?node-id=0%3A1
